README.md

```diff
@@ -1,70 +1,139 @@
+    The page will reload when you make changes.
+    You may also see any lint errors in the console.
+```
 
-## Learn More
+### Setup
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Clone this repository to your desired folder:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+```sh
+  cd my-folder
+  git clone https://github.com/dselasea/react-capstone.git
+```
 
-### Code Splitting
+### Install
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Install this project with:
 
-### Analyzing the Bundle Size
+```sh
+  npm install
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Run tests
 
-### Making a Progressive Web App
+To run tests, run the following command:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+```sh
+  npm test
+```
 
-### Advanced Configuration
+### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Github Pages
+- Netlify
+- Render
 
-### Deployment
+### Future Features
+
+- Crypto Charts
+- Sign Up and Sign In
+- Price ticker
+
+### Live Demo Link
+
+- Live Demo Link: [Live Link](https://dselaseacrypto.netlify.app/)
+
+### Project Video Description
+
+- Video Description: [Video Description of Project](https://www.loom.com/share/ef0dce417d91491693950845be517aab)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Daniel Selase Anyidoho**
+
+- GitHub: [@githubhandle](https://github.com/dselasea)
+- Twitter: [@twitterhandle](https://twitter.com/dselasea)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/dselasea)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for creating this project.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] When I click on your MIt license link in your readme it is broken, please fix it 👍🏿